app/mixins/inplace.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-export default Ember.Mixin.create({
-    i18n: Ember.inject.service(),
+const { computed, inject, Mixin } = Ember;
+const { service } = inject;
```

Comment:
Yes, ES6 provides many new syntax, including destructuring for organization. :+1: